Tools/GenerateWebsiteData.ps1

```diff
@@ -77,6 +79,58 @@ function GetLinks ($docObj) {
     return $null
 }
 
+# Generate a PSObject that will be used to populate the cmdlet JSON file based on $parameter.
+function New-DynamicParamObject($parameter) {
```

Comment:
Is this function name consistent with the rest of the code? Later we define `AddDynamicParameterToDocObj`. Should that instead be `Add-DynamicParameterToDocObj`?